processing/data_collection/gazette/settings.py

```diff
@@ -13,3 +13,4 @@
 # skip already crawled item
 DELTAFETCH_ENABLED = True
 DELTAFETCH_DIR = "/mnt/data/deltafetch"
+MEDIA_ALLOW_REDIRECTS = True
```

Comment:
Thanks for your research! But I would like to ask you one last thing. Move this configuration for the spider. You can see how to do it [here](https://docs.scrapy.org/en/latest/topics/settings.html#settings-per-spider). As most of the spider work without this, I would like to keep this confirm just for this spider. Later, when we face more spider with this issue, we can move the setting to all spider.

Reply:
No problem, done.